packages/styling/components/highlight-box.scss

```diff
@@ -36,14 +36,43 @@ $dso-highlight-box-margin-top: $dso-shared-equal-heights-highlight-box-height;
 .dso-highlight-box,
 dso-highlight-box {
   .btn-link:not([disabled]) {
-    color: $zwart;
+    color: $grasgroen;
+
+    &:hover,
+    &:focus {
+      &,
+      dso-icon,
+      svg.di {
+        color: $scampi;
+      }
+    }
+  }
+}
+
+.dso-highlight-box:not(.dso-border):not(.dso-drop-shadow),
+dso-highlight-box:not([border]):not([drop-shadow]) {
+  .btn-link:not([disabled]) {
+    color: $grijs-90;
 
     &:hover,
     &:focus {
       &,
       dso-icon,
       svg.di {
         color: $grijs-60;
+      }
+    }
+  }
+}
+
+.dso-highlight-box,
+dso-highlight-box {
+  .btn-link:not([disabled]) {
+    &:hover,
+    &:focus {
+      &,
+      dso-icon,
+      svg.di {
         text-decoration: underline;
```

Comment:
Deze had eigenlijk ook wel  gewoon naar regel 56 gekund toch? Zo'n compleet eigen stijl-eiland is volgens mij niet nodig.